apps/src/templates/sectionProgress/progressTables/ProgressTableContentView.jsx

```diff
@@ -100,19 +104,24 @@ export default class ProgressTableContentView extends React.Component {
    * constrain column widths to the provided values. When it's absent, the
    * columns will size themselves based on their content.
    *
-   * Note: Due to the nuances of reactabular's implementation, header cells
+   * One exception is that we provide a fixed width for empty columns when a
+   * lesson has no levels.
+   *
+   * Note: due to the nuances of reactabular's implementation, header cells
    * are unable to base their width on the content of body cells, nor
    * vice versa. Consequently, for headers to properly align with their body
    * columns when explicit column widths are not provided, the max content
    * width of header cells must match the max content width of body cells.
    */
   columnWidthStyle(index) {
-    const {columnWidths} = this.props;
-    return columnWidths
-      ? {
-          style: {minWidth: columnWidths[index], maxWidth: columnWidths[index]}
-        }
-      : {};
+    const {columnWidths, scriptData} = this.props;
+    let width = null;
+    if (columnWidths) {
+      width = columnWidths[index];
+    } else if (!scriptData.stages[index].levels?.length) {
```

Comment:
Since you created a helper for this, you could use it here `!lessonHasLevels(scriptData.stages[index])`

Reply:
👍